sdubov/aws-toolkit-jetbrains
# Add completion to Lambda handler in run configuration editor

- Extend handler completion logic to run per language and runtime group
- Add logic to find all possible lambda handler in a solution to provide them in a completion lookup

## Types of changes

## Description
Show a completion popup for available lambda handlers inside local lambda run configuration.

## Motivation and Context
Simplify process of composing lambda handler string when setup a lambda run configuration.

## Related Issue(s)
None

## Testing
- Check completion select all possible lambdas from different projects
- Check completion results

## Screenshots (if appropriate)
<img width="723" alt="LambdaCompletion" src="https://user-images.githubusercontent.com/6064345/64356541-bb790900-d00b-11e9-8129-e1d9571d0a7f.png">

## Checklist

## License
I confirm that my contribution is made under the terms of the Apache 2.0 license.
